src/Examine.Test/Examine.Lucene/Search/FacetFluentApiTests.cs

```diff
@@ -0,0 +1,2678 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using Examine.Lucene;
+using Examine.Lucene.Providers;
+using Examine.Lucene.Search;
+using Examine.Search;
+using Lucene.Net.Analysis.En;
+using Lucene.Net.Analysis.Standard;
+using Lucene.Net.Facet.Range;
+using Lucene.Net.QueryParsers.Classic;
+using Lucene.Net.Search;
+using NUnit.Framework;
+
+
+
+namespace Examine.Test.Examine.Lucene.Search
+{
+    [TestFixture]
+    public class FacetFluentApiTests : ExamineBaseTest
+    {
+
+        /*
+         * Below is a copy of the tests from FluentApiTests
```

Comment:
Hrm, I don't love the full duplication. I think it would be a lot simpler to maintain this if we used TestCase's instead. So instead of using the `[Test]` attribute, we'd use:

```cs
[TestCase(true)]
[TestCase(false)]
public void Allow_Leading_Wildcards(bool withFacets)
````

and then use the `withFacets` parameter to either search with or without.

Reply:
I'll refactor the tests to use `TestCase` instead 👍

Reply:
This has now been refactored.